### DISCUSSION TITLE
How to Get currently connected wallet's Ethereum Balance

### DISCUSSION THREAD

**Discussion by rachna-khatnawlia** [WalletConnect/walletconnect-monorepo#1786] [Developer Support]
How to Get wallet's Balance of currently connected wallet

**Comment by finessevanes** [comment]
@rachna-khatnawlia are you building a dapp or wallet? What packages are you using?

**Reply by rachna-khatnawlia** [reply]
@finessevanes  Building a BlockChain app using React Native Which is connected to Wallets i.e MetaMask

**Reply by finessevanes** [reply]
@rachna-khatnawlia nice! Do you know what `@web3modal/...` package you're using? https://docs.walletconnect.com/2.0/web3modal/about

ie, Standalone, React, or HTML?

**Reply by rachna-khatnawlia** [reply]
@finessevanes  using @walletconnect/react-native-dapp package for wallet connectivity 
https://docs.walletconnect.com/1.0/quick-start/dapps/react-native

**Reply by rachna-khatnawlia** [reply]
@finessevanes using @walletconnect/react-native-dapp package for wallet connectivity
https://docs.walletconnect.com/1.0/quick-start/dapps/react-native

**Comment by NesredinJemal** [comment]
I need my balance contact account wallet

**Comment by NesredinJemal** [comment]
> How to Get wallet's Balance of currently connected wallet

USD

**Comment by finessevanes** [comment]
@rachna-khatnawlia  What web3 library are you using?
